cdap-ui/templates/cdap-etl-realtime/Stream.json

```diff
@@ -22,8 +22,8 @@
         },
         "name": {
           "widget": "stream-selector",
-          "label" : "Name",
-          "description" : "Name of the stream to which we write to. If the stream is present, then data will be written to it. If it's not present then one would be created.",
+          "label" : "Stream Name",
```

Comment:
When users see this, they're already configuring a Stream source. Do we need to mention Stream again?